runtime/development/src/lib.rs

```diff
@@ -1824,11 +1824,11 @@ construct_runtime!(
 		Nfts: pallet_nft::{Pallet, Call, Event<T>} = 103,
 		Keystore: pallet_keystore::{Pallet, Call, Storage, Event<T>} = 104,
 		Investments: pallet_investments::{Pallet, Call, Storage, Event<T>} = 105,
-		Rewards: pallet_rewards::<Instance1>::{Pallet, Storage, Event<T>} = 106,
+		Rewards: pallet_rewards::<Instance1>::{Pallet, Storage, Event<T>, Config<T>} = 106,
```

Comment:
Out of curiosity, what capabilities add this `Config<T>` here?, something related to the Genesis?

Reply:
This is required for the `GenesisConfig` builder applied in `[src/chain_spec.rs](https://github.com/centrifuge/centrifuge-chain/pull/1342/files#diff-8eaf99be7c59a06fae89dfafbab2ec3873044be161513bb28863e3c99af34d69R1146)`. Without this setting, we would not expose the Config and thus could not do any genesis setting.